.github/workflows/core-contracts-storage-check.yml

```diff
@@ -28,6 +29,22 @@ jobs:
             contracts:
               - packages/contracts/src/dollar/core/*.sol
 
+      - name: Remove newly added contracts from the list
+        id: filter-new-contracts
+        run: |
+          touch contracts.txt
+          CHANGED_CONTRACTS="${{ steps.changed-contracts.outputs.contracts_all_changed_files }}"
+          
+          # Iterate through changed contracts and check if they existed in the previous commit
+          for CONTRACT in $CHANGED_CONTRACTS; do
+            if git show HEAD~1:$CONTRACT > /dev/null 2>&1; then
```

Comment:
Check this example for a PR branch:
1. Commit 1: docs updated
2. Commit 2: new contract `MyContract` is added (ci checked that `MyContract` didn't exist in the previous commit and skipped the storage check)
3. Commit 3: docs updated again (ci checked that `MyContract` existed in the previous commit and run storage check while the expected behavior is to skip it)

Here on step 3 ci will be still failing because there's no such contract in the default `development` branch.